types/types.go

```diff
@@ -28,6 +31,35 @@ type Conf struct {
 	Cron        string      `yaml:"cron"`
 }
 
+func (conf Conf) MissingCronSpec() bool {
+	return conf.Cron == ""
+}
+
+func ParseCronSpec(spec string) cron.Schedule {
+	sched, err := cron.ParseStandard(spec)
+
+	if err != nil {
+		log.Error().Str("spec", spec).Msg(err.Error())
+	}
+
+	return sched
+}
+
+func (conf Conf) HasValidCronSpec() bool {
+	if conf.MissingCronSpec() {
+		return false
+	}
+
+	parsedSched := ParseCronSpec(conf.Cron)
+
+	if parsedSched != nil {
+		nextRun := parsedSched.Next(time.Now()).String()
+		log.Info().Str("next", nextRun).Msg("Next cron run")
```

Comment:
What I really want here, and the whole reason to get into this, is to get a human-readable cron expression.

Could I bring in https://github.com/lnquy/cron to do this? It's not a scheduler, just a cron-ese to human converter.